site/write_ups/2022/MarginalisedGroups.md

```diff
@@ -0,0 +1,78 @@
+2.	Any focus on genetics ignores a whole host of social and societal factors which cannot be neglected as part of a complex picture. 
+3.	Many researchers continue to frame autism entirely under the medical model of disability (hence the use of medical databases), and completely ignores the social model of disability which is a far more appropriate framework for properly understanding autism and other neurodiverse neurotypes.
+
+## Systematic bias in the data
+
+Autism is commonly under- and mis-identified in several groups in society. The reasons for this include historical biases dating back to the earliest identification of the neurotype - even today the myth that “autism is only found in boys” is pervasive. This is compounded by historical biases in identifying autism in anyone who isn’t a white boy conforming to a very [narrow range of behaviour stereotypes](https://pubmed.ncbi.nlm.nih.gov/30143373/). 
+Assumptions based on outdated and wrong information have contributed to diagnostic bias against [women, non-binary people, and anyone of colour](https://pubmed.ncbi.nlm.nih.gov/26544750/).
+
+## Social and societal factors
+
+These historical biases are amplified by other social and societal issues. Autistic people report that one of the biggest challenges to thriving in a society designed around neurotypical social norms is the constant pressure to “mask” their autism by conforming to the expected norms of society which [neurotypical people take for granted](https://www.liebertpub.com/doi/abs/10.1089/aut.2021.0021).  
+
+One reason for the under-diagnosis of autistic women is that societal pressures are much stronger on people socialised as women to behave in certain ways. As a result, it has been estimated that as much as [80% of autistic women are unidentified as autistic at age 18](https://pubmed.ncbi.nlm.nih.gov/35204992/). Far from the 1:4 ratio of autistic women to autistic men that are currently recognised by existing diagnoses, attempts to estimate the true ratio have found it is at least 3:4. Again due to historical biases of diagnostic criteria, these estimates are by definition not exact, but equally there seems to be [no evidence as to why the ratio shouldn’t be 1:1](https://www.scientificamerican.com/article/we-need-better-diagnostic-tests-for-autism-in-women/).  
+
+## Medical vs Social models of disability 
+
+A proper discussion of different models of disability is beyond the scope of this post  . To summarise, the medical model of disability says that a disabled person has something “wrong” with them, to be “fixed”. The [social model] (https://inclusionscotland.org/get-informed/social-model) recognises that a disabled person might have different support needs in order to fully participate in society, but since society does not currently provide such support, the disabled person is in effect, disabled by society.
```

Comment:
```suggestion
A proper discussion of different models of disability is beyond the scope of this post. To summarise, the medical model of disability says that a disabled person has something “wrong” with them, to be “fixed”. The [social model] (https://inclusionscotland.org/get-informed/social-model) recognises that a disabled person might have different support needs in order to fully participate in society, but since society does not currently provide such support, the disabled person is in effect, disabled by society.
```